File: Orion after party/ThreeScore.java
/* PSUEDOCODE
1. START
2. CREATE A REQUEST FOR THREE SCORE FROM THE USER
3. ADD THE SUM OF THE SCORES TOGETHER
4. CALCULATE THE AVERAGE (TOTAL SUM / THREE) 
5. USE THE RESULT TO PRINT OUT THE GRADE OF THE SCORE USING THE IF STATEMENT.
6. END 
*/

import java.util.Scanner;
public class ThreeScore{
public static void main(String [] args) {
Scanner input = new Scanner(System.in);
	
	int average = 0;
	int sum = 0;

	for(int reader = 1; reader <=3; reader++) {
	System.out.print("Enter score: ");
	int score = input.nextInt();

	sum = sum + score;
	average = sum / 3;
		}
	
	if(average >=90 && average <= 100){
	System.out.print("The grade is A");
}
	if(average>= 80 && average < 90){
	System.out.print("The grade is B");
}
	if(average>= 70 && average <80){
	System.out.print("The grade is C");
}
	if(average >= 60 && average < 70){
	System.out.print("The grade is D");
}
	if(average>= 0 && average < 60){
	System.out.print("The grade is F");
}

	
		}

	}	
	

File: Orion after party/three_score.py
# PSUEDOCODE
# START
# CREATE A REQUEST FOR THREE SCORE FROM THE USER
# ADD THE SUM OF THE SCORES TOGETHER
# CALCULATE THE AVERAGE (TOTAL SUM / THREE) 
# USE THE RESULT TO PRINT OUT THE GRADE OF THE SCORE USING THE IF EL STATEMENT.
# END


score_one = int(input("Enter score one"))
score_two = int(input("Enter score two"))
score_three = int(input("Enter score three"))

sum=0
sum = score_one +score_two + score_three
print (sum)
average = sum / 3
print (average)

if(90 <= average <= 100):
	print("The grade is A")

elif(80<= average < 90):
	print("The grade is B")

elif(70 <= average <80):
	print("The grade is C")

elif(60 <= average < 70):
	print("The grade is D")

elif(0 <= average < 60):
	print("The grade is F")


